### PR TITLE
Beacd 76

### PR DESCRIPTION
**Pull request by gaurav-hmrc**

### CODE REVIEW

**Comment by gaurav-hmrc** [comment]
This pull request includes BEACD-54

File: conf/messages.cy [hunk 1]
```diff
@@ -670,3 +701,22 @@ registerInHomeCountry.heading = Cofrestru yn eich gwlad wreiddiol
 registerInHomeCountry.p1=Dylech gofrestru ar gyfer GUC TAW gyda’r awdurdod treth yng ngwlad wreiddiol eich busnes.
 registerInHomeCountry.p2=Os ydych eisiau cofrestru ar gyfer GUC TAW yn y DU, mae angen i chi
 registerInHomeCountry.callUs = ein ffonio.
+
+#######################################################
+##  AreYouContractorOrSubcontractor
+#######################################################
+areYouContractorOrSubcontractor.title = e]: Ydych chi’n gontractwr neu’n isgontractwr?
```
Comment (kurtlogan) [inline conf/messages.cy]: is `e]:` supposed to be in here?

File: conf/messages.cy [hunk 2]
```diff
@@ -670,3 +701,22 @@ registerInHomeCountry.heading = Cofrestru yn eich gwlad wreiddiol
...
+areYouContractorOrSubcontractor.heading = Ydych chi’n gontractwr neu’n isgontractwr?
+areYouContractorOrSubcontractor.Contractor = Contractwr - yn cyflogi ac yn talu isgontractwyr
+areYouContractorOrSubcontractor.Subcontractor = Isgontractwr - yn gweithio i fusnesau adeiladu eraill
+areYouContractorOrSubcontractor.error.required = Dewiswch p’un a ydych yn gontractwr neu’n isgontractwr
+areYouContractorOrSubcontractor.p1=Os ydych y ddau, bydd rhaid i chi wneud hyn ddwywaith. Dewiswch opsiwn, yna dychwelwch a dewiswch y llall.
+
+#######################################################
+##  IsYourBusinessInUK
+#######################################################
+isYourBusinessInUK.title = A yw eich busnes wedi'i leoli yn y DU?
```
Comment (kurtlogan) [inline conf/messages.cy]: Missed apostrophe

File: conf/messages.cy [hunk 3]
```diff
@@ -670,3 +701,22 @@ registerInHomeCountry.heading = Cofrestru yn eich gwlad wreiddiol
...
+isYourBusinessInUK.heading = A yw eich busnes wedi'i leoli yn y DU?
```
Comment (kurtlogan) [inline conf/messages.cy]: Missed apostrophe

File: conf/messages.cy [hunk 4]
```diff
@@ -670,3 +701,22 @@ registerInHomeCountry.heading = Cofrestru yn eich gwlad wreiddiol
...
+isYourBusinessInUK.Yes = Ydy - mae fy musnes wedi'i leoli yn y DU
```
Comment (kurtlogan) [inline conf/messages.cy]: Missed apostrophe

File: conf/messages.cy [hunk 5]
```diff
@@ -670,3 +701,22 @@ registerInHomeCountry.heading = Cofrestru yn eich gwlad wreiddiol
...
+isYourBusinessInUK.No = Nac ydy - nid yw fy musnes wedi'i leoli yn y DU
```
Comment (kurtlogan) [inline conf/messages.cy]: Missed apostrophe

File: test/views/employer/cis/IsYourBusinessInUKViewSpec.scala [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+      }
+    }
+
+    for (option <- IsYourBusinessInUK.options) {
+      s"rendered with a value of '${option.value}'" must {
+        s"have the '${option.value}' radio button selected" in {
+          val doc = asDocument(createViewUsingForm(form.bind(Map("value" -> s"${option.value}"))))
+          assertContainsRadioButton(doc, option.id, "value", option.value, true)
+
+          for (unselectedOption <- IsYourBusinessInUK.options.filterNot(o => o == option)) {
```
Comment (kurtlogan) [inline test/views/employer/cis/IsYourBusinessInUKViewSpec.scala]: Replace lambda with `_ == option`. This has been changed in the template now

File: test/views/employer/cis/ukbased/AreYouContractorOrSubcontractorViewSpec.scala [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      }
+    }
+
+    for (option <- AreYouContractorOrSubcontractor.options) {
+      s"rendered with a value of '${option.value}'" must {
+        s"have the '${option.value}' radio button selected" in {
+          val doc = asDocument(createViewUsingForm(form.bind(Map("value" -> s"${option.value}"))))
+          assertContainsRadioButton(doc, option.id, "value", option.value, true)
+
+          for (unselectedOption <- AreYouContractorOrSubcontractor.options.filterNot(o => o == option)) {
```
Comment (kurtlogan) [inline test/views/employer/cis/ukbased/AreYouContractorOrSubcontractorViewSpec.scala]: replace lambda with `_ == option`

**Pull request merged**